eng/common/scripts/Update-DocsMsMetadata.ps1

```diff
@@ -165,5 +180,12 @@ function UpdateDocsMsMetadataForPackage($packageInfoJsonLocation) {
 
 foreach ($packageInfo in $PackageInfoJsonLocations) {
   Write-Host "Updating metadata for package: $packageInfo"
+  # Add validation step for daily update and release
+  if ($ValidateDocsMsPackagesFn -and (Test-Path "Function:$ValidateDocsMsPackagesFn")) {
+    &$ValidateDocsMsPackagesFn -PackageInfo $packageInfo 
```

Comment:
We should pass in the PackageSourceOverride here as well. Even if someone doesn't provide that parameter it should be a no-op for those.

Reply:
Then we have to add check null step here before making the call. My preference is to check null right before the logic.

Reply:
We don't need to add a null check we just blindly pass it and if any consumers need it, they will declare the parameter and they will check it before using it.

Reply:
Do you mean we pass something like:

```
&$ValidateDocsMsPackagesFn -PackageInfo $packageInfo  $PackageSourceOverride DocValidationImageId
```

If we pass something like below, poweshell will complain taking null object
```
&$ValidateDocsMsPackagesFn -PackageInfo $packageInfo  -PackageSourceOverride  $PackageSourceOverride -DocValidationImageId $DocValidationImageId
```

Reply:
This script is the only place to call ValidateDocsMsPackagesFn currently.

Reply:
I would expect the following to work:
```
&$ValidateDocsMsPackagesFn -PackageInfo $packageInfo  -PackageSourceOverride  $PackageSourceOverride -DocValidationImageId $DocValidationImageId
```
What error are you seeing?